src/EnergyPlus/ScheduleManager.hh

```diff
@@ -186,10 +197,14 @@ namespace ScheduleManager {
         bool MaxMinSet;                         // Max/min values have been stored for this schedule
         Real64 MaxValue;                        // Maximum value for this schedule
         Real64 MinValue;                        // Minimum value for this schedule
-        Real64 CurrentValue;                    // For Reporting
-        bool EMSActuatedOn;                     // indicates if EMS computed
-        Real64 EMSValue;                        // EMS value
-        bool UseDaylightSaving;                 // Toggles between daylight saving option to be inclused as "No" or "Yes" (default)
+        std::array<bool, static_cast<int>(DayType::Num)> MaxMinByDayTypeSet{
+            false}; // minimum and maximum values by daytype have been stored for this schedule
+        std::array<Real64, static_cast<int>(DayType::Num)> MinByDayType{0.0}; // minimum values by daytype for this schedule
+        std::array<Real64, static_cast<int>(DayType::Num)> MaxByDayType{0.0}; // maximum values by daytype for this schedule
```

Comment:
Multiple arrays allocated to the same size hints that they _may_ be nicer as a single struct.  Not a big deal since there are just 3 arrays, but if it grew to more, it might be worthwhile.